aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -842,6 +849,29 @@ module aptos_framework::coin {
         });
     }
 
+    #[test(account = @0x1)]
+    public fun test_is_coin_store_frozen(account: signer) acquires CoinStore {
+        let account_addr = signer::address_of(&account);
+        account::create_account_for_test(account_addr);
+        let (burn_cap, freeze_cap, mint_cap) = initialize_and_register_fake_money(&account, 18, true);
```

Comment:
Do you also want a test for the uninitialized account case?

Reply:
Yes please

Reply:
There is a test for that in the latest commit

Reply:
Awesome!!